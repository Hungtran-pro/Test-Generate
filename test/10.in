29
1252
1822
1909
1665
1498
1728
1471
1616
1776
1552
1598
1789
1331
1435
1338
1000
1899
1675
1073
1437
1353
1818
1926
1034
1291
1147
1519
1008
1027
