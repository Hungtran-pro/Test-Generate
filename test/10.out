147591088
195059542
530065956
114040082
573888282
271366137
828155164
142284658
532945912
667766160
597811311
687555330
460588245
554680665
291756657
184260998
138833480
219357158
316086847
36985926
919725318
874332347
343552022
97871345
539298801
134858755
726395268
192773580
147848872
